CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-### [1.1.0](https://github.com/by-pinja/ngx-resource-calendar/compare/1.0.0...1.1.0) (2023-xx-xx)
+### [1.1.0](https://github.com/by-pinja/ngx-resource-calendar/compare/1.0.0...1.1.0) (2023-10-04)
```

Comment:
I think that it's wiser to introduce version `2.0.0` of this library when upgrading major version of Angular itself.